ow_faults_injection/cfg/Faults.cfg

```diff
@@ -100,7 +100,7 @@ ft_sensor_faults.add("signal_noise_failure", double_t, 0,
 
 power_faults = gen.add_group("power_faults")
 power_faults.add("high_power_draw",   double_t, 0,
-    "High power draw increment", 1.0, 1.0, 20.0)
+    "High power draw increment", 1.0, 1.0, 500.0)
```

Comment:
I suggest making this consistent with the 90W limit, i.e. 360, until the final values are determined.  This can be changed after this PR.